tests/core/scene/a-scene.test.js

```diff
@@ -28,6 +28,21 @@ suite('a-scene (without renderer)', function () {
     test('initializes scene object', function () {
       assert.equal(this.el.object3D.type, 'Scene');
     });
+
+    test('does not initialize systems', function () {
+      var sceneEl = document.createElement('a-scene');
+      assert.notOk(Object.keys(sceneEl.systems).length);
+    });
+  });
+
+  suite('attachedCallback', function () {
+    test('initializes systems', function (done) {
+      var self = this;
+      process.nextTick(function () {
```

Comment:
Instead of doing next tick here it's probably more clear to wait for `loaded` in the setup function like we do here 
https://github.com/aframevr/aframe/blob/master/tests/components/geometry.test.js#L13